src/components/StatsGrid/StatsGridConfig.tsx

```diff
@@ -114,6 +163,17 @@ export const columnDefs = [
   {
     field: "ip",
     headerName: "IP & ISP",
+    tooltipComponent: HTMLTooltip,
+    headerTooltip: (() => {
+      return [
+        htmlBold("IP address and ISP of the node."),
+        "IP address is obsured and not available publicly to protect the node from direct access.",
+        htmlSeparator(),
+        htmlItalic(
+          "This data is currently provided by an external service and can be inaccurate - we are working to remedy this situation."
```

Comment:
```suggestion
          "This data is currently provided by an external service and can be inaccurate. Please report any mismatch in #filecoin-saturn Slack channel"
```

Reply:
Actually, do we really care about ISP exactness?

Reply:
I don't think so, it's strictly informational. What we really care is the physical location mismatches.